Services/Calendar/classes/class.ilCalendarSchedule.php

```diff
@@ -80,6 +85,8 @@ public function __construct(ilDate $seed,$a_type,$a_user_id = 0, $a_strict_perio
 
 	 	$ilUser = $DIC['ilUser'];
 	 	$ilDB = $DIC['ilDB'];
+
+	 	$this->logger = $DIC->logger()->calc();
```

Comment:
The id of the calendar service is "cal". see Services/Calendar/service.xml -> id
Therefore it should be logger()->cal()

Reply:
Yes, fixed.